src/lib/storage/lz4_segment.cpp

```diff
@@ -448,7 +448,7 @@ std::shared_ptr<AbstractSegment> LZ4Segment<T>::copy_using_allocator(const Polym
 }
 
 template <typename T>
-size_t LZ4Segment<T>::memory_usage(const MemoryUsageCalculationMode) const {
+size_t LZ4Segment<T>::memory_usage(const MemoryUsageCalculationMode /*mode*/) const {
```

Comment:
`unused`?

Reply:
Same here: consistency with parent class and all classes that actually use the `mode.`.